scripts/setup-opam.sh

```diff
@@ -2,6 +2,12 @@
 set -x # echo on
 set -eu
 
+# Make sure this is called from root directory
+if [[ "$(pwd)" != "$(git rev-parse --show-toplevel)" ]]; then
```

Comment:
In most bash scripts we instead handle changing directories based on the path the script is stored at, so that the script works consistently but we no longer care where it is executed from. See https://github.com/MinaProtocol/mina/blob/f88edb440e321114e26f7691e599adab30ce16cd/automation/scripts/upload-keys-k8s.sh#L13 for an example

Reply:
in our case we do care where it is executed from though, as we're setting up the root directory. We could in theory set it up from a different directory, but we'd have to change all the commands to accommodate that which can be pretty verbose.